### DISCUSSION TITLE
backup and restore su ssd

### DISCUSSION THREAD

**Discussion by Domconnor** [indomus/forum#2814] [GENERALE - Discussioni generiche]
allora scusate la domanda da niubbo
in pratica ho installato e configurato il home assistant os su un raspberry 4 con micro sd

poi con pratica e miglioramenti vari ho iniziato a leggere varie guide(..) ok lo so si doveva fae al contrario

quindi ora voglio comprare un ssd e installare docker e ha sopra.. è possibile farlo usando il mio backup con tutta la mia configurazione ??
o cosa è meglio fare ?

grazie

**Comment by snakuzzo** [comment]
No...puoi recuperare la configurazione di core e riportarla sull anuova installazione (poi dovrai controllare eventuali piccole cose da correggere).
Per quanto riguarda gli addon...come saprai dopo aver letto le varie modalità di installazione, dovrai tirare su servizi a parte come altri container docker e poi dovrai riportare "a mano" le configurazioni che avevi prima per gli addon

**Comment by Domconnor** [comment]
grazie della risposta 
speravo fosse più semplice.. dovrò smanettare più del previsto

**Reply by snakuzzo** [reply]
Diciamo che non è complicato. Se non sei pratico, ovviamente potresti impiegarci più del previsto. Però devi sempre considerare che non è detto che sia semplice e veloce così com'è detto che non sia difficile e lungo. Va valutato a seconda dei casi... in base a quanti addon hai, se ci sono immagini docker ufficiali/pronte al posto degli addon, e capire poi se ci sono eventuali configurazioni particolari da fare.

**Comment by 50494554524F** [comment]
tieni haos, aggiungi l'ssd... sposta la partizione dati su quest'ultimo, lo puoi fare dalla ui... la microsd tanto la userà solo all'avvio

**Reply by snakuzzo** [reply]
A meno che non abbia capito male io, lui ha chiesto in merito al passaggio a Core su Docker...che c'entra la tua risposta? Hai letto solo il titolo?

**Reply by 50494554524F** [reply]
hai capito male tu

> o cosa è meglio fare ?

**Reply by snakuzzo** [reply]
Ah...in tal senso sì, se non vuole cambiare tipo di installazione ok.
Se invece vuole passare a core su docker (che, se sai un minimo smanettare secondo me è meglio per tutta una serie di motivi già indicati su chat, Discussions e articoli sul sito), vale la risposta che gli ho dato prima.

**Comment by Domconnor** [comment]
grazie per la risposta molto interessante potrei iniziare così e vedere come procede la cosa 

poi quando divento più "competente" posso ricreare tutto

gentilissimi
